src/org.xtuml.bp.core/models/org.xtuml.bp.core/ooaofooa/Functions/Functions.xtuml

```diff
@@ -817,6 +818,22 @@ INSERT INTO PE_PE
 	"2467e6dc-9ef4-46c6-afa6-883c1e23a96c",
 	"00000000-0000-0000-0000-000000000000",
 	1);
+INSERT INTO S_SYNC
```

Comment:
We already have a modeled activity to return null instances. We should not introduce a second. It is GD::NULL_INSTANCE(). (Graphics Domain EE)

Reply:
Agreed, removed and used the existing operation